Bot/functions.py

```diff
@@ -1,40 +1,192 @@
 import random
+import re
 import time
 from tkinter import Label
-from typing import Callable, List
+from typing import Any, Callable, List, Literal, NamedTuple
 
+import PIL
 import pyautogui
+import pytesseract  # type: ignore
 from loguru import logger
+import platform
+
+if platform.system() == "Windows":
+    pytesseract.pytesseract.tesseract_cmd = "C:\\Program Files\\Tesseract-OCR\\tesseract.exe"
```

Comment:
make this a property ? in the dist files we could preconfigure this property with for ex ".\tesseract.exe" since it will most probably exist in the zip file.

Reply:
not sure though how this can be done for macOS, but primary use case windows anyway so